Makefile

```diff
@@ -51,6 +52,21 @@ MD_FILES := $(shell \
 		-name '*.md' \
 	| sed 's/ /\\ /g' \
 )
+UI_FILES = $(shell \
```

Comment:
Grabbing the dependencies with static analysis is that fast that I can include the rest now, but before yes, it was for performance reasons that I excluded them.

In the tests and docs [examples] case it is for stats reasons, I don't think that we should count the component usage in tests and docs.

I will see if I can come up with an automated way to filter out them.